### ISSUE TITLE
iOS Exception, need to cal WidgetsFlutterBinding.ensureInitialized() 

### ISSUE THREAD

**Issue by louwers** [gdelataillade/alarm#281]
Exception has occurred.
_AssertionError ('package:flutter/src/services/platform_channel.dart': Failed assertion: line 554 pos 7: '_binaryMessenger != null || BindingBase.debugBindingType() != null': Cannot set the method call handler before the binary messenger has been initialized. This happens when you call setMethodCallHandler() before the WidgetsFlutterBinding has been initialized. You can fix this by either calling WidgetsFlutterBinding.ensureInitialized() before this or by passing a custom BinaryMessenger instance to MethodChannel().)

The example on the README throws an exception. `WidgetsFlutterBinding.ensureInitialized() ` needs to be called before Alarm.init()

**Comment by gdelataillade** [comment]
Yes you must call `WidgetsFlutterBinding.ensureInitialized();` before calling `Alarm.init` in the `main` function. I'll add it to the README.
